iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/NodeBreakerVoltageLevel.java

```diff
@@ -166,6 +166,8 @@ public Switch add() {
             }
             SwitchImpl aSwitch = new SwitchImpl(NodeBreakerVoltageLevel.this, id, getName(), kind, open, retained, fictitious);
             getNetwork().getIndex().checkAndAdd(aSwitch);
+            addNodes(node1);
```

Comment:
I think I will rename this method to `reserveNodes` or `ensureNode`. The **s** in the name of the method seems strange for someone who reads the code. You have to go in the implementation to understand why you put an **s**.